grafana/k6-studio
# feat: Error Boundary

## Description

Add an error element for the top-level route error boundary

## How to Test

* Create a correlation rule
* Select `Begin-end` in the extractor type
* Type `\` in the `Begin` input field - you will see the error view
* Click `Go back` and verify that you've been redirected to the home view

## Checklist



## Screenshots (if appropriate):
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/317660ae-9915-49f0-9401-684197cc01b4">
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/26e688cb-d98a-4825-a002-c0a1016ef078">


## Related PR(s)/Issue(s)
